application-workloads/dokku/dokku-vm/azuredeploy.json

```diff
@@ -82,7 +80,7 @@
   "variables": {
     "storageAccountName": "[concat(uniquestring(resourceGroup().id), 'dokku')]",
     "imagePublisher": "Canonical",
-    "imageOffer": "UbuntuServer",
+    "imageOffer": "0001-com-ubuntu-server-focal",
```

Comment:
can you expand on the need for this change?

Reply:
@bmoore-msft I updated the version of Ubuntu to 20.04. In order to do that, the image offer also needed to be changed and the default has changed from Ubuntu Server to the focal image.

Here is some more information: https://github.com/Azure/azure-cli/issues/13320#issuecomment-649867249

Reply:
Got it - thanks...